src/Rewrite.hs

```diff
@@ -54,16 +55,41 @@ data Args
         derivationContents :: Text
       }
 
+type Rewriter = (Text -> IO ()) -> Args -> ExceptT Text IO (Maybe Text)
+
+type Plan = [(Text, Rewriter)]
+
+standardPlan :: Plan
+standardPlan =
+  [ ("version", version),
+    ("rustCrateVersion", rustCrateVersion),
+    ("golangModuleVersion", golangModuleVersion),
+    ("", quotedUrlsET) -- Don't change the logger
+    --("redirectedUrl", Rewrite.redirectedUrls)
+  ]
+
+updateScriptPlan :: Plan
+updateScriptPlan =
+  [ ("updateScript", updateScript),
+    ("", Rewrite.quotedUrlsET) -- Don't change the logger
+    --("redirectedUrl", Rewrite.redirectedUrls)
+  ]
```

Comment:
Rather than making this a different type of thing entirely, I would just update the other updates to skip if there's an update script -- similar to how `updateVersion` will skip if there's a `cargoSha256` or `goModSha256`.  Seems like then it'd flow more consistently without conditional branching.

Reply:
Good idea, thanks.